src/js/JournalContainer.js

```diff
@@ -46,19 +66,42 @@ const JournalTextEntry = () => {
   };
 
   const saveEntry = () => {
-    let timestamp = Date.now();
+    let entries = [...journalEntries];
+    let timestamp = Date(Date.now());
```

Comment:
This is makes it so the date of edited entries is changed to when it was edited and not when it was first made.
we might want to have a separate value for edited date to preserve the date. The date also does not show up in the journal container, so we need to ask if the design team wants it there.  I would also format the date to look prettier.